webpack/storybook/molecules/FormCard.stories.js

```
@@ -0,0 +1,239 @@
+      ></TextBlock>
+      <TextInput type="text"></TextInput>
+    </LabelFieldPair>,
+    <CheckBox
+      key="checkbox"
+      label={"Same as Property Address"}
+      checked={isSameAsPropertyAddress}
+      onChange={handleCheckboxChange}
+    ></CheckBox>,
+  ];
+
+  return <FormCard {...args} children={updatedChildren} />;
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)